index.html

```diff
@@ -3,28 +3,51 @@
 
 <head>
  <Title>Date Range Analyser</Title>
- <script src="./index.js">
- </script>
+
 
  <style>
```

Comment:
Styles in the same file? Why don't you have another file called styles.css and include it on this page instead as a resource?

Reply:
Styles.css file added to project.